pkg/runner/fedora.go

```diff
@@ -15,5 +15,6 @@ func (p *Fedora) GetBuildPackages() []string {
 		"glibc",   // ldconfig
 		"systemd", // systemd-tmpfiles and systemd-sysusers
 		"python3", // osbuild
+		"dnf",     // temporary
```

Comment:
temporary? What does this mean?

Reply:
Ah yes, good that this was caught. I am wondering if it's correct to add `dnf` to the build packages here.

Reply:
@achilleas-k wdyt?

Reply:
If we're going to be marking packages for all fedora builds, then we'll need it in the build root, yes.